docs/learn-launch.md

```diff
@@ -70,3 +70,13 @@ parachain slot auctions, parathreads, and cross-chain message passing. These upg
 runtime upgrades that will pass through Polkadot's normal governance processes. The core
 functionality does not have to be unlocked sequentially &mdash; several features can be unlocked
 with a single proposal.
+
+## Polkadot 2.0
+
+After the release of Polkadot 1.0, research for the next version of Polkadot will begin. However
+these areas of possible research are subject to change as Polkadot evolves. Possible areas of
+research could include:
+
+- Nested or heirarchical Relay Chains, where Relay Chains could be connected through parachains.
+- Support for three tiers of parachains, each working in parallel to one another.
```

Comment:
Can you clarify this? I don't think I understand, and I haven't heard of this concept before